.github/workflows/tacos_apply.yml

```diff
@@ -113,6 +113,13 @@ jobs:
             jq 'map(.author.login) | join(", ")' -r |
             gha-set-env 'GITHUB_PR_APPROVERS' \
           ;
+          gh pr view \
+              ${{ github.event.pull_request.html_url }} \
+              --json state \
+              --jq .state |
+            gha-set-env 'GITHUB_PR_STATE' \
+          ;
+
```

Comment:
This has gotten non-trivial.
Let's get it into a real bash script.
Perhaps: `lib/ci/github-pr-env`